staff/maryam-malek/cv/index.html

```diff
@@ -4,76 +4,85 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="../cv-styled/style.css">
     <title>Curriculum Vitae</title>
    
 </head>
 <body>
-        <header>
-            <h1>Richard House</h1>
-            <address>
+    <header class="header">
```

Comment:
Checkout the tabulation. Also, using a `<header>` within a `<header>` might not be recommended. [Check out this link](https://stackoverflow.com/questions/16546482/header-inside-a-header-html5) :) Please, correct me if I am wrong.

Reply:
there isn't a header inside another